data/scripts.js

```diff
@@ -2166,8 +2163,6 @@ exports.BattleScripts = {
 			item = template.baseStats.spe >= 60 && template.baseStats.spe <= 108 && !counter['priority'] && this.random(3) ? 'Choice Scarf' : 'Choice Specs';
 		} else if (counter.Special >= 3 && hasMove['uturn'] && template.baseStats.spe >= 60 && template.baseStats.spe <= 108 && !counter['priority'] && this.random(3)) {
 			item = 'Choice Scarf';
-		} else if (hasMove['partingshot'] && !teamDetails.zMove && this.random(2)) {
-			item = 'Darkinium Z';
```

Comment:
Pangoro still has Parting Shot; this change would have affected it and shouldn't have been removed.